cores/arduino/ard_sup/gpio/ap3_gpio.cpp

```diff
@@ -76,32 +86,34 @@ void padMode(uint8_t pad, am_hal_gpio_pincfg_t mode)
 }
 
 // translate Arduino style pin mode function to apollo3 implementation
-void pinMode(uint8_t pin, uint8_t mode) {
+void pinMode(uint8_t pin, uint8_t mode)
+{
 
     am_hal_gpio_pincfg_t pinmode = AP3_GPIO_PINCFG_NULL;
 
-    switch (mode) {
-        case INPUT:
-            pinmode = AP3_PINCFG_INPUT;
-            break;
-        case OUTPUT:
-            pinmode = AP3_PINCFG_OUTPUT;
-            break;
-        case INPUT_PULLUP:
-            pinmode = AP3_PINCFG_INPUT_PULLUP;
-            break;
-        case INPUT_PULLDOWN:
-            pinmode = AP3_PINCFG_INPUT_PULLDOWN;
-            break;
-        case OPEN_DRAIN:
-            pinmode = AP3_PINCFG_OPEN_DRAIN;
-            break;
-        case TRISTATE:
-            pinmode = AP3_PINCFG_TRISTATE;
-            break;
-        default:
-            //no match, just do nothing
-            return;
+    switch (mode)
+    {
+    case INPUT:
+        pinmode = AP3_PINCFG_INPUT;
+        break;
+    case OUTPUT:
+        pinmode = AP3_PINCFG_OUTPUT;
+        break;
+    case INPUT_PULLUP:
+        pinmode = AP3_PINCFG_INPUT_PULLUP;
+        break;
+    case INPUT_PULLDOWN:
+        pinmode = AP3_PINCFG_INPUT_PULLDOWN;
+        break;
+    case OPEN_DRAIN:
+        pinmode = AP3_PINCFG_OPEN_DRAIN;
+        break;
+    case TRISTATE:
+        pinmode = AP3_PINCFG_TRISTATE;
+        break;
+    default:
+        //no match, just do nothing
+        return;
```

Comment:
It was autoformatting, but I figured since we were trying to use the autoformatter more, it would be worth just leaving in.

To clarify, it the preference to have a separate PR with the white space changes or just as separate commit in the same PR?

Reply:
Yes definitely worth keeping the changes. Same PR, different commit. Not a big deal though seriously.